assets/css/global.css

```diff
@@ -48,7 +48,7 @@
     */
     --text: var(--polar-night-dark);
     --text-muted: var(--polar-night-very-light);
-    --background: var(--snow-storm-light);
+    --background: url('/bg/light.svg');
```

Comment:
Wrong path, the css folder is missing!

Reply:
oh... you are supposed to write the name of the folder in which this code is contained? that explains all my bugs with other programs...

Reply:
Here's an example:
![image](https://user-images.githubusercontent.com/60521810/155311097-3ab6c67c-e271-4eff-83db-f11d9ecc87ed.png)

Reply:
oh. also i have to put the ~ mark?
and if the css file (referring to the image) is in the root folder, what would you use?